.circleci/config.yml

```diff
@@ -1,42 +1,128 @@
 version: 2.1
 
-jobs:
-  standards:
+commands:
+  install-dependencies:
```

Comment:
I so see the dashes vs underscores difference between this command and the one you added.  I know this command already existed, you just moved things around.  Part of me says let's pick one way.  I prefer underscores for these commands.  You're call if you change it in this PR.

Reply:
Good call, consistency is important. I'll see if there is a CircleCI preferred standard, if not then I'll standardize one way or the other with reasons for why.

Reply:
Alright, decided to go with `snake_case` as the rest of Circle's default commands follow this pattern. (IE: `attach_workspace` and `persist_to_workspace`)